src/main/java/seedu/eduke8/Command.java

```diff
@@ -0,0 +1,12 @@
+package seedu.eduke8;
+
+import seedu.eduke8.common.DisplayableList;
+import seedu.eduke8.ui.Ui;
+
+
+public interface Command {
```

Comment:
Do you think making this into an abstract class would be better than an interface because other command classes will eventually inherit from it instead of just specifying the behaviour?

Reply:
Done thanks